src/main/java/io/github/antishake/AntiShake.java

```diff
@@ -322,7 +322,7 @@ void setEarliestAccelerometerDataIndex(int earliestAccelerometerDataIndex1) {
       accelerometerValues = new ArrayList<Coordinate>();
     }
     accelerometerValues.clear();
-    accelerometerValues.addAll(Arrays.asList(getCircularBuffer().toArray()));
+    accelerometerValues.addAll(getCircularBuffer().getList());
```

Comment:
Renamed toArray to getList, as toArray was returning an array and now i changed to return an ArrayList. If we change to readAll(). we need to do code changes in convolve and isShaking methods in AntiShake.java.

Lets raise a new issue for next sprint and fix it. Whats your thought?